kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java

```diff
@@ -43,11 +44,12 @@
     EthClientWrapper(android.content.Context androidContext, ServiceProvider serviceProvider)
         throws EthereumClientException {
         this.serviceProvider = serviceProvider;
+        this.androidContext = androidContext.getApplicationContext();
```

Comment:
I prefer not to save the context - there is no need to right?  wipeoutAccount can get context as well

Reply:
we're just saving application context - that is ok